Automattic/wp-calypso
# My Home: Fix alignment on inspiration card call-to-action button

Updates style on call to action button for inspiration card

Fixes #97421

## Proposed Changes

* Updates styles on container,

## Why are these changes being made?


* Fixes #97421
### Before

<img width="551" alt="image" src="https://github.com/user-attachments/assets/2ac9c9d4-3cff-4028-935f-2add41a08360" />


<img width="564" alt="image" src="https://github.com/user-attachments/assets/6b8f386c-d0c7-4efe-8186-c55b46c01564" />


### After 

<img width="594" alt="image" src="https://github.com/user-attachments/assets/e739a5bb-e418-4baa-a6cb-e89d3721c8a2" />
<img width="598" alt="image" src="https://github.com/user-attachments/assets/df6f9637-17df-4281-8dd0-1c441f9bd20e" />

## Testing Instructions

* Make a new site.
* Navigate to my home. (`/home/yoursite.wordpress.com`)
* See fixed alignment for the button.

## Pre-merge Checklist
